mimir-build-image/Dockerfile

```
@@ -5,7 +5,7 @@
 
 FROM k8s.gcr.io/kustomize/kustomize:v4.5.5 as kustomize
 FROM alpine/helm:3.8.2 as helm
-FROM golang:1.19.3-bullseye
+FROM golang:1.20.1
```

Let's be specific about which linux distribution to use. You will need to rebuild the docker image after this change.

```suggestion
FROM golang:1.20.1-bullseye
```

---

Yep agree with @pracucci, there's no need to drop the distro specifier.

---

my appologies to `go go` :)